iwyu.cc

```diff
@@ -250,6 +250,8 @@ bool CanIgnoreLocation(SourceLocation loc) {
 
 }  // anonymous namespace
 
+enum class UseKind { Direct, RemoveRefs, RemoveRefsAndPtr };
```

Comment:
`UseKind` is a little open-ended, but I like it because it's short. Maybe `DerefKind` could work?

Reply:
`Direct` should probably be replaced with `DerefKind::None` in that case?

Reply:
Sounds good!